library/billing.inc

```diff
@@ -216,9 +214,6 @@ function updateClaim($newversion, $patient_id, $encounter_id, $payer_id=-1, $pay
           "pid = '$patient_id' AND encounter = '$encounter_id'");
       }
     }
```

Comment:
Be careful.  This setting has 3 cases, and only 1 of them (1) is the sql-ledger case.  There is also the case (0) of no A/R support, and I'm not sure if that is catered to anywhere.